api/types/types.proto

```diff
@@ -261,6 +261,9 @@ message DatabaseSpecV3 {
     DatabaseTLS TLS = 8 [ (gogoproto.nullable) = false, (gogoproto.jsontag) = "tls,omitempty" ];
     // AD is the Active Directory configuration for the database.
     AD AD = 9 [ (gogoproto.nullable) = false, (gogoproto.jsontag) = "ad,omitempty" ];
+    // Options are additional database options.
+    DatabaseOptions Options = 10
```

Comment:
Same comment as in fileconfig, I'd probably prefer having a `mysql` subsection with `server_version` field.

Reply:
Changed